toolsWebsite/tools/templates/tools/available.html

```diff
@@ -50,12 +50,28 @@
                     <i class="fa fa-cart-arrow-down"></i>
                   </a>
                   <div class="dropdown-menu dropdown-menu-right" aria-labelledby="cartDropdown">
-                    <div id="cartDropdownItems"></div>
-                    <a href="/checkout">
-                      <button class="btn btn-outline-info my-2 my-sm-0 right shoppingDropdownButton">
-                        Check Out
-                      </button>
-                    </a>
+                    <b>Items in Cart</b>
+                    <div id="cartDropdownItems">
+                      <div class="tool-container">
```

Comment:
I think we should put this `div` inside the for loop here so it separates tools.

Reply:
We can do that.  Do any of you know how we could write it so the navbar is an import?  That way we don't have to change it in every file when we want to change it.

Reply:
I can look it up again, but I'm still trying to figure it out. We can fix this later anyways, I'll just merge this in, and we can talk about it later.